components/canvas/backend.rs

```diff
@@ -115,7 +116,8 @@ pub(crate) trait GenericDrawTarget<B: Backend> {
         draw_options: &B::DrawOptions,
     );
     fn surface(&self) -> B::SourceSurface;
-    fn bytes(&self) -> Cow<[u8]>;
+    fn wr_data(&self) -> (ImageDescriptor, SerializableImageData);
```

Comment:
Let's improve the name a little:

```suggestion
    fn image_descriptor_and_serializable_data(&self) -> (ImageDescriptor, SerializableImageData);
```